DRAFT_gjengroing.Rmd

```diff
@@ -0,0 +1,994 @@
+  
+  # Calcualte gjengroing index values
+  vaatmarkIndexPoly <- vaatmarkHts  %>%
+    # Gather into long format so taht we can mutate the scaleSigmoid() function
+    gather(type, height, ref, pop, skog) %>%
+    group_by(`system:index`) %>%
+    mutate(index = scaleSigmoid(height)) %>%
+    # Because short vegetation is good condition, we must invert the scaled indicator
+    mutate(index = 1-index) %>%
+    # Get the indicator values for the population polygons only (ref and skog will be 1 and 0)
+    filter(type == 'pop') %>%
+    dplyr::select(-type, -height) %>%
+    # Join with original data to get the actual median vegetaiton heights for each indicator score
+    left_join(vaatmarkHts %>% dplyr::select("system:index", pop, ref, skog)) %>%
+    left_join(popVaatmark %>% dplyr::select("system:index", ssbid))
+  
+  # do the same for åpne polygons - will not comment this because same as above
+  aapneIndexPoly <- aapneHts  %>%
+    gather(type, height, ref, pop, skog) %>%
+    group_by(`system:index`) %>%
+    mutate(index = scaleSigmoid(height)) %>%
+    mutate(index = 1-index) %>%
+    filter(type == 'pop') %>%
+    dplyr::select(-type, -height) %>%
+    left_join(aapneHts %>% dplyr::select("system:index", pop, ref, skog)) %>%
+    left_join(popAapne %>% dplyr::select("system:index", ssbid))
+  
+  
+  # Make into spatial objects
+  vaatmarkIndexPolySpat <- popVaatmark %>%
+    dplyr::select("system:index") %>%
+    left_join(vaatmarkIndexPoly) %>%
+    filter(!is.na(index))
+  aapneIndexPolySpat <- popAapne %>%
+    dplyr::select("system:index") %>%
+    left_join(aapneIndexPoly) %>%
+    filter(!is.na(index))
+  
+  # Transform to correct projection
+  vaatmarkIndexPolySpat <- st_transform(vaatmarkIndexPolySpat, st_crs(regions))
+  aapneIndexPolySpat <- st_transform(aapneIndexPolySpat, st_crs(regions))
+  
+} else {
+  
+  vaatmarkIndexPolySpat <- st_read('P:/41201785_okologisk_tilstand_2022_2023/gjengroing/R_project/OUTPUT/vaatmark_index.shp')%>%
+    mutate(`system:index` = id,
+           elevation = elev, 
+           vegClimZoneLab = vegZn) %>%
+    dplyr::select(-id, -elev, -vegZn)
+  aapneIndexPolySpat <- st_read('P:/41201785_okologisk_tilstand_2022_2023/gjengroing/R_project/OUTPUT/aapne_index.shp')%>%
+    mutate(`system:index` = id,
+           elevation = elev, 
+           vegClimZoneLab = vegZn) %>%
+    dplyr::select(-id, -elev, -vegZn)
+  
+}
+
+
+```
+
+Here we will visualize the spatial variation in scaled gjengroind indicator values, as well as the vegetation heights for reference (good condition), indicator, and mature forest (poor condition) over the country. We could plot all the polygons, but it is computationally intensive and very difficult to visualize spatial variations over the entire country. Therefore we will calculate the area-weighted mean in values for each bioclimatic-elevation strata (n = 21) over the country.
+
+```{r, fig.width=10}
+
+# Create intersection of the two strata layers
```

Comment:
I get stuck here (I will try and fix now) - Error is "Error in geos_op2_geom("intersection", x, y, ...) : 
  st_crs(x) == st_crs(y) is not TRUE" : which is correct = <html>
<body>
<!--StartFragment--><div style="position: absolute; overflow: hidden; top: 0px; right: 0px; bottom: 0px; width: 805px;"><div style="width: 805px; height: 935.6px; position: absolute; inset: 0px;"><div style="position: relative; width: 805px; height: 935.6px;"><div style="position: absolute; overflow: hidden; z-index: 0; inset: 0px 0px 632px;"><div role="region" aria-label="Console" class="windowframe GND-IWGDGW GND-IWGDMQ" style="-webkit-font-smoothing: subpixel-antialiased; position: absolute; inset: 0px;"><div style="border: 1px solid rgb(214, 218, 220); border-radius: 3px 3px 0px 0px; inset: 3px 3px 5px; overflow: hidden; position: absolute;"><div style="position: absolute; inset: 0px;"><div style="position: absolute; overflow: hidden; inset: 0px;"><div class="gwt-TabLayoutPanel GND-IWGDJS gwt-TabLayoutPanel-Workbench" style="background: rgb(244, 248, 249); position: absolute; inset: 0px; width: 803.4px; height: 302px;"><div style="inset: 23px 0px 0px; position: absolute; overflow: hidden;"><div class="gwt-TabLayoutPanelContentContainer" style="position: absolute; inset: 0px;"><div style="position: absolute; overflow: hidden; left: 0px; top: 0px; width: 803.4px; height: 279px;"><div id="rstudio_workbench_panel_console" role="tabpanel" aria-label="Console" class="gwt-TabLayoutPanelContent" style="background: rgb(244, 248, 249); position: absolute; inset: 0px;"><div style="position: relative; width: 803.4px; height: 279px;"><div style="position: absolute; overflow: hidden; inset: 0px;"><div style="position: absolute; width: 803.4px; height: 279px; inset: 0px;"><div style="position: absolute; overflow: hidden; inset: 22px 0px 0px;"><div style="width: 803.4px; height: 257px; position: absolute; inset: 0px;"><div class="ace_editor ace_scroller" id="rstudio_shell_widget" style="position: relative; overflow: auto; font-style: normal; font-variant: normal; font-kerning: auto; font-optical-sizing: auto; font-feature-settings: normal; font-variation-settings: normal; font-weight: normal; font-stretch: normal; font-size: 12px; line-height: normal; font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace !important; direction: ltr; text-align: left; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); top: 0px; bottom: 0px; background-color: rgb(255, 255, 255); user-select: none; cursor: text; border: none !important; color: black; zoom: 1; width: 786.6px; height: 257px;"><div style="position: relative; zoom: 1;">

> st_crs(bioclim) Coordinate Reference System:   User input: WGS 84 / UTM zone 33N    wkt: PROJCRS["WGS 84 / UTM zone 33N",     BASEGEOGCRS["WGS 84",         DATUM["World Geodetic System 1984",             ELLIPSOID["WGS 84",6378137,298.257223563,                 LENGTHUNIT["metre",1]]],         PRIMEM["Greenwich",0,             ANGLEUNIT["degree",0.0174532925199433]],         ID["EPSG",4326]],     CONVERSION["UTM zone 33N",         METHOD["Transverse Mercator",             ID["EPSG",9807]],         PARAMETER["Latitude of natural origin",0,             ANGLEUNIT["Degree",0.0174532925199433],             ID["EPSG",8801]],         PARAMETER["Longitude of natural origin",15,             ANGLEUNIT["Degree",0.0174532925199433],             ID["EPSG",8802]],         PARAMETER["Scale factor at natural origin",0.9996,             SCALEUNIT["unity",1],             ID["EPSG",8805]],         PARAMETER["False easting",500000,             LENGTHUNIT["metre",1],             ID["EPSG",8806]],         PARAMETER["False northing",0,             LENGTHUNIT["metre",1],             ID["EPSG",8807]]],     CS[Cartesian,2],         AXIS["(E)",east,             ORDER[1],             LENGTHUNIT["metre",1]],         AXIS["(N)",north,             ORDER[2],             LENGTHUNIT["metre",1]],     ID["EPSG",32633]] > st_crs(elevStrata) Coordinate Reference System:   User input: WGS 84    wkt: GEOGCRS["WGS 84",     DATUM["World Geodetic System 1984",         ELLIPSOID["WGS 84",6378137,298.257223563,             LENGTHUNIT["metre",1]],         ID["EPSG",6326]],     PRIMEM["Greenwich",0,         ANGLEUNIT["degree",0.0174532925199433],         ID["EPSG",8901]],     CS[ellipsoidal,2],         AXIS["geodetic longitude",east,             ORDER[1],             ANGLEUNIT["degree",0.0174532925199433]],         AXIS["geodetic latitude",north,             ORDER[2],             ANGLEUNIT["degree",0.0174532925199433]]]
--
 
> | > |  
> |  

</div><div style="position: absolute; overflow: hidden; inset: 46px 0px 16px;"><div class="ace_editor_theme" style="background-color: rgb(255, 255, 255); color: black; position: absolute; inset: 0px;"><div aria-hidden="true" style="position: absolute; z-index: -32767; top: -20ex; width: 10em; height: 10ex; visibility: hidden;"></div><div style="position: absolute; overflow: hidden; inset: 0px;"><div style="position: absolute; inset: 0px;"><div aria-hidden="true" style="position: absolute; z-index: -32767; top: -20ex; width: 10em; height: 10ex; visibility: hidden;"></div><div style="position: absolute; overflow: hidden; top: 0px; right: 0px; bottom: 0px; width: 110px;"><div class="GND-IWGDMNC rstudio_doc_outline_container" style="width: 110px; height: 416px; position: absolute; inset: 0px; background-color: rgb(255, 255, 255); color: rgb(0, 0, 0);"><div aria-hidden="true" style="position: absolute; z-index: -32767; top: -20ex; width: 10em; height: 10ex; visibility: hidden;"></div><div style="position: absolute; overflow: hidden; padding-right: 4px; left: 0px; top: 0px; bottom: 0px; width: 4px;"><div class="GND-IWGDONC" role="separator" aria-orientation="vertical" style="border-left: 1px solid rgba(128, 128, 128, 0.5); cursor: col-resize; width: 8px; z-index: 1; position: absolute; inset: 0px;"></div></div><div style="position: absolute; overflow: hidden; inset: 0px 0px 0px 4px;"><div class="GND-IWGDGOC" style="overflow-y: auto; position: absolute; inset: 0px;"><div class="gwt-Tree GND-IWGDHOC" aria-label="Document Outline" style="overflow: hidden auto; padding-top: 3px; padding-bottom: 3px; position: relative; zoom: 1;"><div tabindex="0" hidefocus="true" role="tree" style="font-size: 0px; position: absolute; outline: 0px; z-index: -1;"><input type="text" tabindex="-1" aria-hidden="true" style="font-family: &quot;Lucida Sans&quot;, &quot;DejaVu Sans&quot;, &quot;Lucida Grande&quot;, &quot;Segoe UI&quot;, Verdana, Helvetica, sans-serif, serif; padding-left: 4px; opacity: 0; height: 1px; width: 1px; z-index: -1; overflow: hidden; position: absolute;"></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1826" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Introduction</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1827" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">About the underlying data</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1828" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Representativity in time and space</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1829" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Original units</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1830" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Temporal coverage</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1831" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Aditional comments about the dataset</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1832" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Ecosystem characteristic</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1833" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Norwegain standard</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1834" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Collinearities with other indicators</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1835" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Reference state and values</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1836" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Reference state</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1837" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Reference values, thresholds for defining *good ecological condition*, minimum and/or maximum values</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1838" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Uncertainties</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1839" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">References</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1840" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;"></div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Analyses</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1841" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Data sets</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1842" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">    </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Regions</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1843" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">    </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">SSB 10 and 50km grids</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1844" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">    </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Bioclimatic regions</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1845" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">    </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">NiN polygons</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1846" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">    </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Data from Google Earth Engine</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1847" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Explore data coverages</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1848" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Bioclimatic zone and elevation strata for regional reference values</div></div></div></div><div class="GND-IWGDPNC GND-IWGDKNC" style="padding: 3px 3px 3px 23px; border-left: 2px solid rgb(103, 137, 10); margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1849" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Scaled indicator values at polygon level (circa 120 min runtime)</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1850" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Scaled indicator values at 50km grid level (circa 45 min runtime)</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1851" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Scaled indicator values at regional level (circa 120 min runtime)</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1852" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Uncertainty and validation</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1853" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">    </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Validation of random polygons</div></div></div></div><div class="GND-IWGDPNC" style="padding: 3px 3px 3px 23px; margin-left: 0px;"><div class="gwt-TreeItem" role="treeitem" id="gwt-uid-1854" style="display: inline;"><div><div class="gwt-HTML GND-IWGDAOC" style="white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden; float: left;">  </div><div class="gwt-Label GND-IWGDAOC GND-IWGDFR" style="cursor: pointer; white-space: nowrap; font-family: sans-serif; font-size: 0.9em; text-overflow: ellipsis; overflow: hidden;">Eksport file (final product)</div></div></div></div></div></div></div></div></div><div style="position: absolute; overflow: hidden; inset: 0px 110px 0px 0px;"><div class="gwt-HTML GND-IWGDOW ace_editor" id="rstudio_source_text_editor_0" draggable="false" style="position: absolute; overflow: hidden; font-style: normal; font-variant: normal; font-kerning: auto; font-optical-sizing: auto; font-feature-settings: normal; font-variation-settings: normal; font-weight: normal; font-stretch: normal; font-size: 10.4pt !important; line-height: 1.1; font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace !important; direction: ltr; text-align: left; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); border: none !important; background-color: rgb(255, 255, 255); color: black; width: 1054px; height: 416px; inset: 0px;"><textarea class="ace_text-input" wrap="off" autocorrect="off" autocapitalize="off" spellcheck="false" aria-label="Text editor" style="outline: none; position: absolute; z-index: 0; width: 1px; height: 1px; opacity: 0; background: transparent; appearance: none; border: none; resize: none; overflow: hidden; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; font-size: 1px; line-height: inherit; font-family: inherit; font-optical-sizing: inherit; font-kerning: inherit; font-feature-settings: inherit; font-variation-settings: inherit; padding: 0px 1px; margin: 0px -1px; contain: strict; user-select: text; white-space: pre !important; orphans: 2; widows: 2; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; top: 179px; left: 60px;"></textarea><div class="ace_gutter" aria-hidden="true" style="position: absolute; overflow: hidden; width: 56px; top: 0px; bottom: 0px; left: 0px; cursor: default; z-index: 4; user-select: none; contain: size layout style; background: rgb(240, 240, 240); color: rgb(51, 51, 51); font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace; font-size: 13.8667px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="ace_layer ace_gutter-layer ace_folding-enabled" style="z-index: 3; position: relative; overflow: hidden; overflow-wrap: normal; white-space: pre; height: 1e+06px; width: 56px; box-sizing: border-box; pointer-events: auto; text-align: right; contain: size layout style; top: -12301px; left: 0px;"><div class="ace_gutter-cell  ace_chunk-queued-line ace_chunk-executed-line" style="position: absolute; top: 12030px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 300px;">591<span class="ace_fold-widget ace_end ace_open" style="box-sizing: border-box; margin: 0px -12px 0px 1px; display: inline-block; width: 11px; vertical-align: top; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANElEQVR42m3HwQkAMAhD0YzsRchFKI7sAikeWkrxwScEB0nh5e7KTPWimZki4tYfVbX+MNl4pyZXejUO1QAAAABJRU5ErkJggg==&quot;); background-repeat: no-repeat; background-position: center center; border-radius: 3px; border: 1px solid transparent; cursor: pointer; height: 15px;"></span></div><div class="ace_gutter-cell " style="position: absolute; top: 12330px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">592</div><div class="ace_gutter-cell " style="position: absolute; top: 12345px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 60px;">593</div><div class="ace_gutter-cell " style="position: absolute; top: 12405px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">594</div><div class="ace_gutter-cell " style="position: absolute; top: 12420px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">595<span class="ace_fold-widget ace_start ace_open" style="box-sizing: border-box; margin: 0px -12px 0px 1px; display: inline-block; width: 11px; vertical-align: top; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANElEQVR42mWKsQ0AMAzC8ixLlrzQjzmBiEjp0A6WwBCSPgKAXoLkqSot7nN3yMwR7pZ32NzpKkVoDBUxKAAAAABJRU5ErkJggg==&quot;); background-repeat: no-repeat; background-position: center center; border-radius: 3px; border: 1px solid transparent; cursor: pointer; height: 15px;"></span></div><div class="ace_gutter-cell " style="position: absolute; top: 12435px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">596</div><div class="ace_gutter-cell " style="position: absolute; top: 12450px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">597</div><div class="ace_gutter-cell  ace_chunk-error-line" style="position: absolute; top: 12465px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">598</div><div class="ace_gutter-cell  ace_chunk-error-line" style="position: absolute; top: 12480px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">599</div><div class="ace_gutter-cell  ace_chunk-error-line" style="position: absolute; top: 12495px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">600</div><div class="ace_gutter-cell  ace_chunk-error-line" style="position: absolute; top: 12510px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">601</div><div class="ace_gutter-cell  ace_chunk-error-line" style="position: absolute; top: 12525px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">602</div><div class="ace_gutter-cell  ace_chunk-error-line" style="position: absolute; top: 12540px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">603</div><div class="ace_gutter-cell " style="position: absolute; top: 12555px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">604</div><div class="ace_gutter-cell " style="position: absolute; top: 12570px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">605<span class="ace_fold-widget ace_start ace_open" style="box-sizing: border-box; margin: 0px -12px 0px 1px; display: inline-block; width: 11px; vertical-align: top; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANElEQVR42mWKsQ0AMAzC8ixLlrzQjzmBiEjp0A6WwBCSPgKAXoLkqSot7nN3yMwR7pZ32NzpKkVoDBUxKAAAAABJRU5ErkJggg==&quot;); background-repeat: no-repeat; background-position: center center; border-radius: 3px; border: 1px solid transparent; cursor: pointer; height: 15px;"></span></div><div class="ace_gutter-cell " style="position: absolute; top: 12585px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">606</div><div class="ace_gutter-cell " style="position: absolute; top: 12600px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">607</div><div class="ace_gutter-cell " style="position: absolute; top: 12615px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">608</div><div class="ace_gutter-cell " style="position: absolute; top: 12630px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">609</div><div class="ace_gutter-cell " style="position: absolute; top: 12645px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">610</div><div class="ace_gutter-cell " style="position: absolute; top: 12660px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">611</div><div class="ace_gutter-cell " style="position: absolute; top: 12675px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">612</div><div class="ace_gutter-cell " style="position: absolute; top: 12690px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">613</div><div class="ace_gutter-cell " style="position: absolute; top: 12705px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">614</div><div class="ace_gutter-cell " style="position: absolute; top: 12720px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">615</div><div class="ace_gutter-cell " style="position: absolute; top: 12735px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">616</div><div class="ace_gutter-cell " style="position: absolute; top: 12750px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">617</div><div class="ace_gutter-cell " style="position: absolute; top: 12765px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">618</div><div class="ace_gutter-cell " style="position: absolute; top: 12780px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">619</div><div class="ace_gutter-cell " style="position: absolute; top: 12795px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">620</div><div class="ace_gutter-cell " style="position: absolute; top: 12810px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">621</div><div class="ace_gutter-cell " style="position: absolute; top: 12825px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">622</div><div class="ace_gutter-cell " style="position: absolute; top: 12840px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">623</div><div class="ace_gutter-cell " style="position: absolute; top: 12855px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">624</div><div class="ace_gutter-cell " style="position: absolute; top: 12870px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">625</div><div class="ace_gutter-cell " style="position: absolute; top: 12885px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">626</div><div class="ace_gutter-cell " style="position: absolute; top: 12900px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">627</div><div class="ace_gutter-cell " style="position: absolute; top: 12915px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">628</div><div class="ace_gutter-cell " style="position: absolute; top: 12930px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">629</div><div class="ace_gutter-cell " style="position: absolute; top: 12945px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 30px;">630</div><div class="ace_gutter-cell " style="position: absolute; top: 12975px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 30px;">631</div><div class="ace_gutter-cell " style="position: absolute; top: 13005px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">632</div><div class="ace_gutter-cell " style="position: absolute; top: 13020px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">633</div><div class="ace_gutter-cell " style="position: absolute; top: 13035px; left: 0px; right: 0px; padding-left: 19px; padding-right: 13px; background-repeat: no-repeat; height: 15px;">634<span class="ace_fold-widget ace_start ace_open" style="box-sizing: border-box; margin: 0px -12px 0px 1px; display: inline-block; width: 11px; vertical-align: top; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANElEQVR42mWKsQ0AMAzC8ixLlrzQjzmBiEjp0A6WwBCSPgKAXoLkqSot7nN3yMwR7pZ32NzpKkVoDBUxKAAAAABJRU5ErkJggg==&quot;); background-repeat: no-repeat; background-position: center center; border-radius: 3px; border: 1px solid transparent; cursor: pointer; height: 15px;"></span></div></div></div><div class="ace_scroller" style="position: absolute; overflow: hidden; inset: 0px 17px 0px 55.8675px; background-color: rgb(255, 255, 255); user-select: none; cursor: default; color: rgb(0, 0, 0); font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace; font-size: 13.8667px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="ace_content" style="position: absolute; box-sizing: border-box; min-width: 100%; contain: size layout style; top: -271px; left: 0px; width: 981.133px; height: 731px;"><div class="ace_layer ace_print-margin-layer" style="z-index: 2; position: absolute; overflow: hidden; overflow-wrap: normal; white-space: pre; height: 731px; width: 981.138px; box-sizing: border-box; pointer-events: none;"><div class="ace_print-margin" style="position: absolute; height: 731px; width: 1px; background: rgb(232, 232, 232); left: 614px; visibility: hidden;"></div></div><div class="ace_layer ace_marker-layer" style="z-index: 1; position: absolute; overflow: hidden; overflow-wrap: normal; white-space: pre; height: 731px; width: 981.138px; box-sizing: border-box; pointer-events: none;"><div class="ace_foreign_line background_highlight" title="Open Link (Shift+Click)" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 450px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 465px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 480px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 495px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 555px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 570px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 585px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 600px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 615px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 630px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 645px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 660px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 675px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 690px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 0px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 510px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 525px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 540px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 405px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 420px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 435px; left: 0px; right: 0px;"></div><div class="ace_foreign_line background_highlight" style="position: absolute; z-index: -1; background-color: rgb(243, 243, 243); height: 15px; top: 390px; left: 0px; right: 0px;"></div><div class="ace_selection ace_br1 ace_start" style="border-radius: 2px; position: absolute; z-index: 5; background: rgb(181, 213, 255); box-shadow: white 0px 0px 3px 0px; height: 15px; right: 0px; top: 435px; left: 4px;"></div><div class="ace_selection ace_br12" style="border-bottom-right-radius: 3px; border-bottom-left-radius: 3px; position: absolute; z-index: 5; background: rgb(181, 213, 255); height: 15px; width: 175.317px; top: 510px; left: 4px;"></div><div class="ace_selection" style="position: absolute; z-index: 5; background: rgb(181, 213, 255); height: 60px; right: 0px; top: 450px; left: 4px;"></div></div><div class="ace_layer ace_text-layer" style="z-index: 3; position: absolute; overflow: hidden; overflow-wrap: normal; white-space: pre; height: 1e+06px; width: 1e+06px; box-sizing: border-box; pointer-events: none; contain: size layout style; font-style: inherit !important; font-variant: inherit !important; font-weight: inherit !important; font-stretch: inherit !important; font-size: inherit !important; line-height: inherit !important; font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace !important; font-optical-sizing: inherit !important; font-kerning: inherit !important; font-feature-settings: inherit !important; font-variation-settings: inherit !important; margin: 0px 4px; top: -12030px; left: 0px;"><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12030px; left: 0px; right: 0px; height: 300px;"><div class="ace_line" style="height: 15px;"><span class="ace_support ace_function ace_codeend" style="color: rgb(60, 76, 114);">```</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12330px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12345px; left: 0px; right: 0px; height: 60px;"><div class="ace_line" style="height: 15px;">Here we will visualize the spatial variation in scaled gjengroind indicator values, as well as the vegetation heights for </div><div class="ace_line" style="height: 15px;">reference (good condition), indicator, and mature forest (poor condition) over the country. We could plot all the polygons, but </div><div class="ace_line" style="height: 15px;">it is computationally intensive and very difficult to visualize spatial variations over the entire country. Therefore we will </div><div class="ace_line" style="height: 15px;">calculate the area-weighted mean in values for each bioclimatic-elevation strata (n = 21) over the country.</div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12405px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12420px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"><span class="ace_support ace_function ace_codebegin" style="color: rgb(60, 76, 114);">```{r, fig.width=10}</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12435px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12450px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"><span class="ace_comment" style="color: rgb(76, 136, 107);"># Create intersection of the two strata layers</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12465px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"><span class="ace_identifier">bioClimElev</span> <span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">&lt;-</span> <span class="ace_identifier">bioclim</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12480px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_identifier">st_intersection</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">elevStrata</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12495px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_identifier">mutate</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">elevation</span> <span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">=</span> <span class="ace_identifier">elevBand</span><span class="ace_punctuation">,</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12510px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">         <span class="ace_identifier">vegClimZoneLab</span> <span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">=</span> <span class="ace_identifier">NAVN</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12525px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_identifier">dplyr</span><span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">::</span><span class="ace_identifier">select</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">elevation</span><span class="ace_punctuation">,</span> <span class="ace_identifier">vegClimZoneLab</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12540px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_identifier">rowid_to_column</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_string" style="color: rgb(3, 106, 7);">"ID"</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12555px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12570px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;"><span class="ace_keyword" style="color: blue;">if</span> <span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">runFromScratch</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">{</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12585px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  </div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12600px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_comment" style="color: rgb(76, 136, 107);"># Get area-weighted mean for reference, forest and population vegetation heights </span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12615px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_comment" style="color: rgb(76, 136, 107);"># for each unique bioclimatic-elevation zone combination</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12630px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">  <span class="ace_identifier">vaatmarkIndexStrata</span> <span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">&lt;-</span> <span class="ace_identifier">bioClimElev</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span> </div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12645px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">    <span class="ace_comment" style="color: rgb(76, 136, 107);">#sample_n(10000) %&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12660px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">    <span class="ace_identifier">st_join</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">vaatmarkIndexPolySpat</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span> </div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12675px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">                <span class="ace_identifier">mutate</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">area</span> <span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">=</span> <span class="ace_identifier">as.numeric</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">st_area</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">geometry</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span> <span class="ace_punctuation">,</span><span class="ace_identifier">st_intersects</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12690px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">    <span class="ace_identifier">group_by</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">ID</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span></div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12705px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">    <span class="ace_identifier">summarise_at</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">vars</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">pop</span><span class="ace_punctuation">,</span> <span class="ace_identifier">ref</span><span class="ace_punctuation">,</span> <span class="ace_identifier">skog</span><span class="ace_punctuation">,</span> <span class="ace_identifier">index</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_punctuation">,</span> <span class="ace_identifier">funs</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">weighted.mean</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_identifier">.</span><span class="ace_punctuation">,</span> <span class="ace_identifier">w</span><span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">=</span><span class="ace_identifier">area</span><span class="ace_punctuation">,</span> <span class="ace_identifier">na.rm</span><span class="ace_keyword ace_operator" style="color: rgb(104, 118, 135);">=</span><span class="ace_constant ace_language" style="color: rgb(88, 92, 246);">TRUE</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span><span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span> </div></div><div class="ace_line_group" style="contain: size layout style; position: absolute; top: 12720px; left: 0px; right: 0px; height: 15px;"><div class="ace_line" style="height: 15px;">                <span class="ace_identifier">as_tibble</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">(</span><span class="ace_paren ace_keyword ace_operator ace_nomatch" style="color: rgb(104, 118, 135);">)</span> <span class="ace_keyword ace_operator ace_infix" style="color: rgb(104, 118, 135);">%&gt;%</span> </div></div></div><div class="ace_layer ace_marker-layer" style="z-index: 1; position: absolute; overflow: hidden; overflow-wrap: normal; white-space: pre; height: 731px; width: 981.138px; box-sizing: border-box; pointer-events: none;"><div class="GND-IWGDIHC ace_start ace_br15" style="border-radius: 3px; position: absolute; background-repeat: repeat-x; background-position: left bottom; background-size: 6px 2px; margin-top: -1px; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAYAAAACCAYAAAB7Xa1eAAAAFUlEQVR42mNgQAKHJ6u8h2FUDhIAAGmIDnePfeRHAAAAAElFTkSuQmCC&quot;); height: 15px; width: 76.225px; top: 315px; left: 423.237px;"></div><div class="GND-IWGDIHC ace_start ace_br15" style="border-radius: 3px; position: absolute; background-repeat: repeat-x; background-position: left bottom; background-size: 6px 2px; margin-top: -1px; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAYAAAACCAYAAAB7Xa1eAAAAFUlEQVR42mNgQAKHJ6u8h2FUDhIAAGmIDnePfeRHAAAAAElFTkSuQmCC&quot;); height: 15px; width: 83.8475px; top: 360px; left: 400.37px;"></div><div class="GND-IWGDIHC ace_start ace_br15" style="border-radius: 3px; position: absolute; background-repeat: repeat-x; background-position: left bottom; background-size: 6px 2px; margin-top: -1px; background-image: url(&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAYAAAACCAYAAAB7Xa1eAAAAFUlEQVR42mNgQAKHJ6u8h2FUDhIAAGmIDnePfeRHAAAAAElFTkSuQmCC&quot;); height: 15px; width: 83.8475px; top: 585px; left: 156.45px;"></div></div><div class="ace_layer ace_cursor-layer ace_hidden-cursors" style="z-index: 3; position: absolute; overflow: hidden; overflow-wrap: normal; white-space: pre; height: 731px; width: 981.138px; box-sizing: border-box; pointer-events: none;"><div class="ace_cursor" style="z-index: 4; position: absolute; box-sizing: border-box; border-left: 2px solid; transform: translateZ(0px); color: black; opacity: 0.2; display: block; top: 435px; left: 4px; width: 8px; height: 15px; animation-duration: 1000ms;"></div></div></div></div><div class="ace_scrollbar ace_scrollbar-v" style="contain: strict; position: absolute; right: 0px; bottom: 0px; z-index: 6; overflow: hidden scroll; top: 0px; color: rgb(0, 0, 0); font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace; font-size: 13.8667px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; width: 22px; margin-bottom: -1px;"><div class="ace_scrollbar-inner" style="position: absolute; cursor: text; left: 0px; top: 0px; width: 22px; height: 19590px;"> </div></div><div style="color: rgb(0, 0, 0); font: inherit; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; height: auto; width: auto; top: 0px; left: 0px; visibility: hidden; position: absolute; white-space: pre; overflow: hidden;"><div style="height: auto; width: auto; top: 0px; left: 0px; visibility: hidden; position: absolute; white-space: pre; font: inherit; overflow: visible;"></div><div style="height: auto; width: auto; top: 0px; left: 0px; visibility: hidden; position: absolute; white-space: pre; font-style: inherit; font-variant: inherit; font-stretch: inherit; font-size: inherit; line-height: inherit; font-family: inherit; font-optical-sizing: inherit; font-kerning: inherit; font-feature-settings: inherit; font-variation-settings: inherit; overflow: visible;"></div></div><div class=" ace_lineWidgetContainer" style="color: rgb(0, 0, 0); font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace; font-size: 13.8667px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; top: -60px; left: 63.49px; right: 17px; position: absolute; z-index: 3;"></div><div class=" ace_lineWidgetContainer" style="color: rgb(0, 0, 0); font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace; font-size: 13.8667px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; top: -30px; left: 63.49px; right: 17px; position: absolute; z-index: 3;"></div><div class=" ace_lineWidgetContainer" style="color: rgb(0, 0, 0); font-family: Consolas, &quot;Lucida Console&quot;, monospace, monospace; font-size: 13.8667px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; top: -2971px; left: 55.8675px; right: 17px; position: absolute; z-index: 3;"><div class="GND-IWGDODD GND-IWGDDU pm-background-color rstudio_chunk" style="user-select: text; cursor: text; padding-bottom: 7px; position: relative; background-color: rgb(255, 255, 255); height: 520px;"><br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html>

Reply:
I think I can just convert CRS of one to the other but I will try and see what happens!